migration-guide/now-to-next.adoc

```diff
@@ -0,0 +1,276 @@
+There is no need to include an additional repository in `pom.xml` or `settings.xml`.
+
+=== Step 2: Generate a New Maven Project using Apache Maven
+
+Now that we have Maven properly configured, we can move onto the next step, which is to generate a new Maven project:
+
+[start=1]
+. In a terminal window, create a new local folder to for the new Maven project, example: `/Users/developer`, and then naviate to that folder.
+
+. Create the project using Quarkus with the Maven commands below, this will create a Quarkus project called `my-sample-decision-service`, which will be versioned `1.0.0-SNAPSHOT`, including the extensions `kogito-quarkus, dmn, resteasy-reactive-jackson, quarkus-smallrye-openapi, quarkus-smallrye-health`.
+This will create a Quarkus DMN project with the OpenAPI components to get the OpenAPI end points easily with health checks when deploying to Kubernetes:
+
++
+[source,console,subs="+attributes"]
+----
+mvn io.quarkus:quarkus-maven-plugin:2.16.7.Final:create \
+    -DprojectGroupId=com.ibm.bamoe.samples \
+    -DprojectArtifactId=my-sample-decision-service \
+    -DprojectVersion=1.0.0-SNAPSHOT \
+    -DplatformGroupId=io.quarkus.platform \
+    -DplatformArtifactId=quarkus-bom \
+    -DplatformVersion={QUARKUS_VERSION_SHORT} \
+    -Dextensions=kogito-quarkus,dmn,resteasy-reactive-jackson,quarkus-smallrye-openapi,quarkus-smallrye-health
+----
+
+[NOTE]
+====
+As you can see from the Maven command above, there are a number of possible `extensions` that can be added to the project in order to add some sort of specific functionality.  You can find more information on available extensions at the following locations:  https://quarkus.io/extensions/[Quarkus Extensions] and https://quarkus.io/extensions/?search-regex=kogito[Kogito Extensions].
+====
+
+When you create this project you should get a bunch of Maven artifacts start to stream in your console that are being pulled and ultimately are left with a console message like below:
+
+[source,console]
+----
+[INFO]
+[INFO] ========================================================================================
+[INFO] Your new application has been created in /Users/developer/my-sample-decision-service
+[INFO] Navigate into this directory and launch your application with mvn quarkus:dev
+[INFO] Your application will be accessible on http://localhost:8080
+[INFO] ========================================================================================
+[INFO]
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time:  24.548 s
+[INFO] Finished at: 2022-09-27T10:22:31-04:00
+[INFO] ------------------------------------------------------------------------
+----
+
+[start=3]
+.  Add the following `properties` to the `pom.xml` file, using the row numbers below as a guide.
```

Comment:
using the row numbers below as a guide -> Remove below

Re-written
using the following numbers as a guide

Reply:
fixed in next commit